client/extensions/woocommerce/woocommerce-services/state/label-settings/reducer.js

```diff
@@ -39,6 +39,9 @@ reducers[ WOOCOMMERCE_SERVICES_LABELS_INIT_FORM ] = (
 			...formMeta,
 			pristine: true,
 			isLoaded: true,
+			user: {
+				...userMeta,
```

Comment:
is userMeta also an object? If yes then do you need destructuring or simple assignment would be sufficient:
`user: userMeta`